examples/subscriptions still broken

examples/subscriptions still broken. BTW there is no package-lock.json file

```
C:\Users\aaron\GitHub\graphql-yoga\examples\subscriptions>yarn start
yarn run v1.22.10
$ ts-node src/index.ts
C:\Users\aaron\GitHub\graphql-yoga\node_modules\ts-node\src\index.ts:750
    return new TSError(diagnosticText, diagnosticCodes);
           ^
TSError: ⨯ Unable to compile TypeScript:
src/index.ts:58:17 - error TS7006: Parameter 'payload' implicitly has an 'any' type.

58       resolve: (payload) => payload,
                   ~~~~~~~
src/index.ts:62:19 - error TS7006: Parameter '_' implicitly has an 'any' type.

62       subscribe: (_, _args, context) =>
                     ~
src/index.ts:62:22 - error TS7006: Parameter '_args' implicitly has an 'any' type.

62       subscribe: (_, _args, context) =>
                        ~~~~~
src/index.ts:62:29 - error TS7006: Parameter 'context' implicitly has an 'any' type.

62       subscribe: (_, _args, context) =>
                               ~~~~~~~
src/index.ts:74:17 - error TS7006: Parameter 'payload' implicitly has an 'any' type.

74       resolve: (payload) => payload,
                   ~~~~~~~
src/index.ts:78:30 - error TS7006: Parameter '_source' implicitly has an 'any' type.

78     incrementGlobalCounter: (_source, _args, context) => {
                                ~~~~~~~
src/index.ts:78:39 - error TS7006: Parameter '_args' implicitly has an 'any' type.

78     incrementGlobalCounter: (_source, _args, context) => {
                                         ~~~~~
src/index.ts:78:46 - error TS7006: Parameter 'context' implicitly has an 'any' type.

78     incrementGlobalCounter: (_source, _args, context) => {
                                                ~~~~~~~

    at createTSError (C:\Users\aaron\GitHub\graphql-yoga\node_modules\ts-node\src\index.ts:750:12)
    at reportTSError (C:\Users\aaron\GitHub\graphql-yoga\node_modules\ts-node\src\index.ts:754:19)
    at getOutput (C:\Users\aaron\GitHub\graphql-yoga\node_modules\ts-node\src\index.ts:941:36)
    at Object.compile (C:\Users\aaron\GitHub\graphql-yoga\node_modules\ts-node\src\index.ts:1243:30)
    at Module.m._compile (C:\Users\aaron\GitHub\graphql-yoga\node_modules\ts-node\src\index.ts:1370:30)
    at Module._extensions..js (internal/modules/cjs/loader.js:1101:10)
    at Object.require.extensions.<computed> [as .ts] (C:\Users\aaron\GitHub\graphql-yoga\node_modules\ts-node\src\index.ts:1374:12)
    at Module.load (internal/modules/cjs/loader.js:937:32)
    at Function.Module._load (internal/modules/cjs/loader.js:778:12)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:76:12) {
  diagnosticText: "\x1B[96msrc/index.ts\x1B[0m:\x1B[93m58\x1B[0m:\x1B[93m17\x1B[0m - \x1B[91merror\x1B[0m\x1B[90m TS7006: \x1B[0mParameter 'payload' implicitly has an 'any' type.\r\n" +
    '\r\n' +
    '\x1B[7m58\x1B[0m       resolve: (payload) => payload,\r\n' +
    '\x1B[7m  \x1B[0m \x1B[91m                ~~~~~~~\x1B[0m\r\n' +
    "\x1B[96msrc/index.ts\x1B[0m:\x1B[93m62\x1B[0m:\x1B[93m19\x1B[0m - \x1B[91merror\x1B[0m\x1B[90m TS7006: \x1B[0mParameter '_' implicitly has an 'any' type.\r\n" +
    '\r\n' +
    '\x1B[7m62\x1B[0m       subscribe: (_, _args, context) =>\r\n' +
    '\x1B[7m  \x1B[0m \x1B[91m                  ~\x1B[0m\r\n' +
    "\x1B[96msrc/index.ts\x1B[0m:\x1B[93m62\x1B[0m:\x1B[93m22\x1B[0m - \x1B[91merror\x1B[0m\x1B[90m TS7006: \x1B[0mParameter '_args' implicitly has an 'any' type.\r\n" +
    '\r\n' +
    '\x1B[7m62\x1B[0m       subscribe: (_, _args, context) =>\r\n' +
    '\x1B[7m  \x1B[0m \x1B[91m                     ~~~~~\x1B[0m\r\n' +
    "\x1B[96msrc/index.ts\x1B[0m:\x1B[93m62\x1B[0m:\x1B[93m29\x1B[0m - \x1B[91merror\x1B[0m\x1B[90m TS7006: \x1B[0mParameter 'context' implicitly has an 'any' type.\r\n" +
    '\r\n' +
    '\x1B[7m62\x1B[0m       subscribe: (_, _args, context) =>\r\n' +
    '\x1B[7m  \x1B[0m \x1B[91m                            ~~~~~~~\x1B[0m\r\n' +
    "\x1B[96msrc/index.ts\x1B[0m:\x1B[93m74\x1B[0m:\x1B[93m17\x1B[0m - \x1B[91merror\x1B[0m\x1B[90m TS7006: \x1B[0mParameter 'payload' implicitly has an 'any' type.\r\n" +
    '\r\n' +
    '\x1B[7m74\x1B[0m       resolve: (payload) => payload,\r\n' +
    '\x1B[7m  \x1B[0m \x1B[91m                ~~~~~~~\x1B[0m\r\n' +
    "\x1B[96msrc/index.ts\x1B[0m:\x1B[93m78\x1B[0m:\x1B[93m30\x1B[0m - \x1B[91merror\x1B[0m\x1B[90m TS7006: \x1B[0mParameter '_source' implicitly has an 'any' type.\r\n" +
    '\r\n' +
    '\x1B[7m78\x1B[0m     incrementGlobalCounter: (_source, _args, context) => {\r\n' +
    '\x1B[7m  \x1B[0m \x1B[91m                             ~~~~~~~\x1B[0m\r\n' +
    "\x1B[96msrc/index.ts\x1B[0m:\x1B[93m78\x1B[0m:\x1B[93m39\x1B[0m - \x1B[91merror\x1B[0m\x1B[90m TS7006: \x1B[0mParameter '_args' implicitly has an 'any' type.\r\n" +
    '\r\n' +
    '\x1B[7m78\x1B[0m     incrementGlobalCounter: (_source, _args, context) => {\r\n' +
    '\x1B[7m  \x1B[0m \x1B[91m                                      ~~~~~\x1B[0m\r\n' +
    "\x1B[96msrc/index.ts\x1B[0m:\x1B[93m78\x1B[0m:\x1B[93m46\x1B[0m - \x1B[91merror\x1B[0m\x1B[90m TS7006: \x1B[0mParameter 'context' implicitly has an 'any' type.\r\n" +
    '\r\n' +
    '\x1B[7m78\x1B[0m     incrementGlobalCounter: (_source, _args, context) => {\r\n' +
    '\x1B[7m  \x1B[0m \x1B[91m                                             ~~~~~~~\x1B[0m\r\n',
  diagnosticCodes: [
    7006, 7006, 7006,
    7006, 7006, 7006,
    7006, 7006
  ]
}
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

--- a/examples/subscriptions/.gitignore
+++ b/examples/subscriptions/.gitignore
@@ -1,0 +1,1 @@
+src/generated/

--- a/examples/subscriptions/codegen.yml
+++ b/examples/subscriptions/codegen.yml
@@ -1,0 +1,7 @@
+overwrite: true
+schema: './src/**/*.ts'
+generates:
+  ./src/generated/graphql.ts:
+    plugins:
+      - 'typescript'
+      - 'typescript-resolvers'

--- a/examples/subscriptions/package.json
+++ b/examples/subscriptions/package.json
@@ -3,7 +3,8 @@
   "private": true,
   "version": "0.0.1-beta.7",
   "scripts": {
-    "start": "ts-node --transpile-only src/index.ts",
+    "postinstall": "graphql-codegen --config codegen.yml",
+    "start": "ts-node src/index.ts",
     "check": "tsc --pretty --noEmit"
   },
   "dependencies": {
@@ -11,5 +12,10 @@
     "graphql": "^16.1.0",
     "ts-node": "10.4.0",
     "typescript": "^4.4.4"
+  },
+  "devDependencies": {
+    "@graphql-codegen/cli": "^2.6.2",
+    "@graphql-codegen/typescript": "^2.4.8",
+    "@graphql-codegen/typescript-resolvers": "^2.6.1"
   }
 }

--- a/examples/subscriptions/src/index.ts
+++ b/examples/subscriptions/src/index.ts
@@ -7,6 +7,7 @@ import {
   map,
   YogaInitialContext,
 } from '@graphql-yoga/node'
+import { Resolvers } from './generated/graphql'
 
 const wait = (time: number) =>
   new Promise((resolve) => setTimeout(resolve, time))
@@ -40,7 +41,15 @@ const typeDefs = /* GraphQL */ `
 
 let globalCounter = 0
 
-const resolvers = {
+const pubSub = createPubSub<{
+  'globalCounter:changed': []
+}>()
+
+interface Context extends YogaInitialContext {
+  pubSub: typeof pubSub
+}
+
+const resolvers: Resolvers<Context> = {
   Query: {
     hello: () => `Hello world!`,
   },
@@ -55,7 +64,7 @@ const resolvers = {
           await wait(1000)
         }
       },
-      resolve: (payload) => payload,
+      resolve: (payload: any) => payload,
     },
     globalCounter: {
       // Merge initial value with source stream of new values
@@ -71,7 +80,7 @@ const resolvers = {
           // map all events to the latest globalCounter
           map(() => globalCounter),
         ),
-      resolve: (payload) => payload,
+      resolve: (payload: any) => payload,
     },
   },
   Mutation: {
@@ -81,14 +90,6 @@ const resolvers = {
       return globalCounter
     },
   },
-}
-
-const pubSub = createPubSub<{
-  'globalCounter:changed': []
-}>()
-
-interface Context extends YogaInitialContext {
-  pubSub: typeof pubSub
 }
 
 const server = createServer<Context, any>({

